plugins/parsers/influx/parser_test.go

```diff
@@ -16,6 +16,12 @@ func Metric(v telegraf.Metric, err error) telegraf.Metric {
 	return v
 }
 
+const (
+	Uint64Overflow uint64 = 9223372036854775808
```

Comment:
Shouldn't this be larger than Uint64Max?  We need to test 18446744073709551616.

Reply:
That was to verify Int64Max+1 properly typed to uint64.  If we attempt to define a const that’s Uint64Max+1 it’d error on compile.